constants.py

```diff
@@ -0,0 +1,50 @@
+import discord
+HELP_REPLY = discord.Embed(title="Ferris Elf help page", color=0xE84611, description="""
+**help** - Send this message
+**info** - Some useful information about benchmarking
+**best _[day]_ _[part]_** - Best times so far for a day
+**submit _[day]_ _[part]_ <attachment>** - Benchmark attached code
+
+If [_day_] and/or [_part_] is ommited, they are assumed to be today and part 1
+
+Message <@{}> for any questions""")
+
+INFO_REPLY = discord.Embed(title="Benchmark information", color=0xE84611, description="""
+When sending code for a benchmark, you should make sure it looks like.
+```rs
+pub fn run(input: &str) -> i64 {
```

Comment:
COMMENT: This should be `impl Display`, no? We should note that you can pass anything that implements Display, *for example* `-> i64` would work, but the main documentation should mention `impl Display`
later docs can also mention the ascii/&[u8] thing, but that is less critical (since here we imply that days with a non number output will break)

Reply:
We should keep this in mind for later stages, we don't have a finalized interface yet and we may move the docs somewhere else.